docs/openlaw-elements/README.md

```diff
@@ -0,0 +1,172 @@
+const App = () => (
+  <OpenLawForm
+    apiClient={apiClient}
+    executionResult={executionResult}
+    parameters={parameters}
+    onChangeFunction={onChange}
+    openLaw={Openlaw}
+    variables={variables}
+  />
+);
+
+ReactDOM.render(<App />, document.getElementById("your-id-here"));
+```
+
+## Required Params
+
+We'll give more detailed information on the OpenLawForm parameters here.
+
+#### apiClient
+
+You will need to authenticate an instance of the apiClient and pass it in to the component to work. For more information on how to do that, see the [REST API and APIClient](/api-client/) portion of the docs.
+
+```
+apiClient={apiClient}
+```
+
+#### executionResult
+
+You will have to pass in the executionResult from a template compiled with the OpenLaw object. See the [Openlaw Object - execute](/openlaw-object/#template) section of the docs for more details.
+
+```
+executionResult={executionResult}
+```
+
+#### parameters
+
+These are the parameters for values that correlate to template variables. Generally you'll manage these in an onChange function throughout the lifecycle of your app.
+
+```
+parameters={parameters}
+```
+
+#### onChangeFunction
+
+This is the method through which you'll handle form changes, as a user types into form fields or makes value changes, the event will fire.
+
+```
+onChangeFunction={onChange}
+```
+
+#### openlaw
+
+This should be an instance of the Openlaw object imported from our js client tools. You can get information on how to install the library at its [github page](https://github.com/openlawteam/openlaw-client) or find more info on its usage at the [Openlaw Object](/openlaw-object/) section here in the docs.
+
+```
+openlaw={openlaw}
+```
+
+#### variables
+
+These are the executed variables retrieved from the compiled template. You can find more information on generating these from the [getExecutedVariables](/openlaw-object/#getexecutedvariables) section of our docs.
+
+```
+variables={variables}
+```
+
+## Optional Parameters
+
+In addition to our regular parameters, we offer support for the following additional parameters.
+
+#### renderSections
+
+This is a custom renderer for changing the look and feel of sections generated with the Openlaw client.
+
+```
+renderSections: ({
+ children: React.Node,
+ section: string,
+}) => React.Node,
+```
+
+#### sectionTransform
+
+If you need to apply transformations to section data on render, this parameter can be used to do so. The transformed data will be passed to `renderSections`.
+
+```
+sectionTransform: (any, number) => {},
+```
+
+#### textLikeInputClass
+
+This will apply a class to all elements that are text-input like including text, email, number, and textarea.
+
+```
+textLikeInputClass: "any-valid-class",
+```
+
+#### unsectionedTitle
+
+This will apply the title to generated sections that have none. If an empty string is provided the title will be unset. The dedfault title is "Miscellaneous".
```

Comment:
```suggestion
This will apply the title to generated sections that have none. If an empty string is provided the title will be unset. The default title is "Miscellaneous".
```